### ISSUE TITLE
AblePlayer breaks other accessible modals present on page

### ISSUE THREAD

**Issue by mably** [ableplayer/ableplayer#611]
Other accessible modals will now be displayed as soon as you have an AblePlayer active on the page as their `aria-hidden=true` attribute is removed by the following code:

`$('body > *').not('.able-modal-overlay').not('.able-modal-dialog').removeAttr('aria-hidden');`

How can we fix this? This code seems pretty intrusive.

Related issue: #252

**Comment by joedolson** [comment]
Seems to me like this would be solved by switching to use the `inert` property instead of using `aria-hidden` to handle the background content. Using `aria-hidden` was really just a hack to deal with the lack of a global property to handle this.

**Comment by joedolson** [comment]
This should be fixed for the next release.
